awkward/util.py

```diff
@@ -197,5 +197,5 @@ class NDArrayOperatorsMixin(object):
         __neg__ = _unary_method(um.negative, 'neg')
         if hasattr(um, "positive"):
             __pos__ = _unary_method(um.positive, 'pos')
-        __abs__ = _unary_method(um.absolute, 'abs')
-        __invert__ = _unary_method(um.invert, 'invert')
+            __abs__ = _unary_method(um.absolute, 'abs')
+            __invert__ = _unary_method(um.invert, 'invert')
```

Comment:
Why only define `__abs__` and `__invert__` if `hasattr(um, "positive")`?

Reply:
This isn't intentional.  I may have done it wrong, but I meant to revert awkward.util to the same version as master (no changes are needed).